Intersect.Client/MonoGame/File Management/MonoContentManager.cs

```diff
@@ -261,6 +279,22 @@ public override void LoadMusic()
                 var filename = items[i].Replace(dir, "").TrimStart(Path.DirectorySeparatorChar).ToLower();
                 mMusicDict.Add(RemoveExtension(filename), new MonoMusicSource(Path.Combine(dir, filename)));
             }
+
+            // If we have a music index file, load from it!
+            if (File.Exists(Path.Combine("resources", "packs", "music.index")))
+            {
+                MusicPacks = new AssetPacker(Path.Combine("resources", "packs", "music.index"), Path.Combine("resources", "packs"));
+                foreach (var item in MusicPacks.FileList)
+                {
+                    if (!mMusicDict.ContainsKey(RemoveExtension(item)))
```

Comment:
!mMusicDict.ContainsKey(RemoveExtension(item)).ToLower()

Gotta check for the existing key in lower case since we're adding the keys as lower case.

Reply:
Hmm, might have to wait and see if that's still the case when Panda is done with his refactor.